src/components/CartItems/CartItem/CartItem.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { ReactComponent as DecreaseQntCounter } from '../../../assets/icons/cartIcons/decrease.svg';
+import { ReactComponent as IncreaseQntIcon } from '../../../assets/icons/cartIcons/increase.svg';
+import { ReactComponent as RemoveItemIcon } from '../../../assets/icons/cartIcons/remove.svg';
+import {
+  fetchPoductUpdate,
+  fetchProductRemoveFromCart,
+} from '../../../axios/cart/requests';
+import { getPriceToFixed } from '../../../helpers/functions';
+import {
+  removeProductFromCart,
+  updateProductInCart,
+} from '../../../store/actions/cart';
+import Button from '../../UI/Button/Button';
+import QuantityLoader from '../../UI/QuantityLoader/QuantityLoader';
+import classes from './CartItem.module.css';
+
+// displaying separate cart items
+
+const CartItem = ({ item }) => {
+  const { id, name, price, pictureUrl, quantity } = item;
+  const modPrice = getPriceToFixed(price, 2);
+  const [qantityLoader, setQuantityLoader] = useState(false);
+  const dispatch = useDispatch();
+
+  const onRemoveHandler = async () => {
+    // eslint-disable-next-line no-restricted-globals
+    const isConfirmed = confirm(
+      `Are you sure to delete ${name} from your order?`
+    );
+
+    if (isConfirmed) {
+      await fetchProductRemoveFromCart(id);
```

Comment:
What happens if `fetchProductRemoveFromCart()` threw an exception?
What the user will see in this case?

Reply:
added a component for handling errors and showing notifications to the user in the next commit https://github.com/maxSedelnikov/react-shopping-app/pull/1/commits/837937c1fc37a6d896fdbc824fb9010078d2d77a